### DISCUSSION TITLE
Add missing releases and features

### DISCUSSION THREAD

**Discussion by Connum** [opentypejs/opentype.js#556] [General]
We should add the missing releases since 1.3.1 to the releases tab. Starting with 2.0.0, it would also be nice to have a changelog and try to not forget to add new features to the README. I think there are some essential things like color font support, extended writing support etc. that aren't even mentioned yet.

**Comment by ILOVEPIE** [comment]
@Connum there's a good github action called release drafter that creates a new draft release every time a new release is added. It then adds a description of the features added in each PR to the draft release's description. Probably the easiest way to maintain a changelog as all we'd have to do is correctly tag each PR.

**Reply by Connum** [reply]
Souns good! Let's keep that in mind!

**Reply by GrzegorzAndrys** [reply]
@Connum Hello, would it be possible to release new version? old one is old and there are new features i care about
